src/components/Tokens/TokenTable/TimeSelector.tsx

```diff
@@ -147,7 +144,9 @@ export default function TimeSelector() {
                 toggleMenu()
               }}
             >
-              <div>{DISPLAYS[time]}</div>
+              <div>
+                {DISPLAYS[time]} <Trans>volume</Trans>
```

Comment:
this is no longer hidden behind a feature flag and is visible with it turned off
<img width="1268" alt="Screenshot 2023-10-05 at 5 08 43 PM" src="https://github.com/Uniswap/interface/assets/11512321/c59c94dc-8db2-4343-8cbd-74770069f368">